static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -59,26 +59,29 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
         <TraceIssueDetailsContainer>
           <NoOverflowDiv>
             <TraceIssueEventTitle>{title}</TraceIssueEventTitle>
-            <TraceIssueEventTransaction>{event.transaction}</TraceIssueEventTransaction>
+            <TraceIssueEventSubtitle>{subtitle}</TraceIssueEventSubtitle>
```

Comment:
Right now, I'm making `subtitle` be `event.transaction` to prevent changing anything but the true source is `culprit` which we don't fetch